packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -224,6 +224,142 @@ curl -v -X POST \
 }
 ```
 
+#### Add Okta Org2Org application
+
+Adds a new Okta Org2Org application to your organization
+
+##### Settings
+
+
+| Parameter          | Description                                             | DataType | Nullable | Unique | Validation                                |
+| ------------------ | ------------------------------------------------------- | -------- | -------- | ------ | ----------------------------------------  |
+| acsURl  | The Source org Assertion Consumer Service URL | String | TRUE    | FALSE  | [URL](http://tools.ietf.org/html/rfc3986) |
```

Comment:
Using the help text for the description
<img width="703" alt="Screen Shot 2020-10-29 at 11 35 25 AM" src="https://user-images.githubusercontent.com/51870451/97617531-1cfff200-19db-11eb-8849-bcd022db6f2c.png">
<img width="703" alt="Screen Shot 2020-10-29 at 11 35 40 AM" src="https://user-images.githubusercontent.com/51870451/97617534-1e311f00-19db-11eb-8264-6fe2cde916b9.png">

Reply:
* You have *acsURl* in the parameter name, but use *acsUrl* in the request example - which is the correct parameter name
* Change the description to read:  The Assertion Consumer Service (ACS) URL of the source org
* Is it ACS UR**L** or ACS UR**I**?

Reply:
it is urL